src/repo/zcl_abapgit_repo.clas.abap

```diff
@@ -305,9 +306,7 @@ CLASS zcl_abapgit_repo IMPLEMENTATION.
 
   METHOD create_new_log.
```

Comment:
do we still need this?

Reply:
No, will fix it in the next commits (to this PR), just didn't have time to finish.

Reply:
fixed